src/App.js

```diff
@@ -26,6 +27,7 @@ function App() {
         <Route element={<ProtectRoutes />}>
           <Route exact path="/" element={<DoctorsList />} />
           <Route exact path="/doctors/:id" element={<DrDetail />} />
+          <Route path="/users/:id/doctors" element={<NewDoctor />} />
```

Comment:
It would have been better if you add an anonymous route to add the doctor instead of showing the current user his id.